src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -976,6 +997,31 @@ export class HaConfigDevicePage extends LitElement {
     this._deviceActions = deviceActions;
   }
 
+  private async _getDeviceAlerts() {
+    const device = this._device(this.deviceId, this.devices);
+
+    if (!device) {
+      return;
+    }
+
+    const deviceAlerts: DeviceAlert[] = [];
+
+    const domains = this._integrations(device, this.entries).map(
+      (int) => int.domain
+    );
+
+    if (domains.includes("zwave_js")) {
+      const zwave = await import(
+        "./device-detail/integration-elements/zwave_js/device-alerts"
+      );
+
+      const alerts = await zwave.getZwaveDeviceAlerts(this.hass, device);
+      deviceAlerts.push(...alerts);
+    }
+
+    this._deviceAlerts = deviceAlerts;
```

Comment:
```suggestion
if (deviceAlerts.length) {
    this._deviceAlerts = deviceAlerts;
}
```
This prevents an unnecessary re-render, as we already set it to empty before calling this function. (should also be done for the others probably)